htdocs/accountancy/bookkeeping/listbyaccount.php

```diff
@@ -144,6 +152,7 @@
 	't.credit'=>array('label'=>$langs->trans("Credit"), 'checked'=>1),
 	't.lettering_code'=>array('label'=>$langs->trans("LetteringCode"), 'checked'=>1),
 	't.date_export'=>array('label'=>$langs->trans("DateExport"), 'checked'=>1),
+	't.date_validated'=>array('label'=>$langs->trans("DateValidation"), 'checked'=>1),
```

Comment:
What is this date ? Is it for a feature of lettering enabled by option ACCOUNTING_ENABLE_LETTERING or by something else ?

Reply:
No, this does not concern the lettering of accounts. It is related to the FEC format, it is requested to validate the accounting entries and to block them in deletion / modification. Without this functionality, a FEC format can be totally validated

![image](https://user-images.githubusercontent.com/2341395/127389256-56758be9-e7fb-4d52-a634-db69d45608cf.png)

Reply:
My question was Where and when can we set this date into application ? On which page ? I can't find. Do we have to set an option or beeing in dev lide to be able to set this date ?

Reply:
This date are set when you export on journal or when you use accoutancy > close menu